eks-anywhere-common/Testers/Nirmata/kyverno-testjob.yaml

```diff
@@ -12,7 +12,8 @@ spec:
           serviceAccountName: 'kyverno'
           containers:
             - name: job
-              image: 'alpine/k8s:1.26.2'
+              # image: 'alpine/k8s:1.26.2'
```

Comment:
The above one had a bunch of vulnerabilities. The [guidelines page](https://aws-samples.github.io/eks-anywhere-addons/#functional-job-requirements) mentioned that image should not have any vulnerabilities.